backendservice/src/main/java/com/google/gpay/virtualqueue/backendservice/model/Token.java

```diff
@@ -18,20 +18,28 @@
 
 import java.util.UUID;
 
-import lombok.AllArgsConstructor;
 import lombok.Data;
+import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 @Data
 @NoArgsConstructor
-@AllArgsConstructor
+@Getter
 public class Token {
     private UUID tokenId;
     private UUID shopId;
     private Integer tokenNumber;
-    
+
     public enum Status {
         STATUS_UNKNOWN, ACTIVE, EXPIRED, CANCELLED_BY_CUSTOMER, CANCELLED_BY_SHOP_OWNER;
     }
+
     private Status status;
+
+    public Token(UUID tokenId, UUID shopId, Integer tokenNumber) {
```

Comment:
Where are we planning to this constructor?

Reply:
yes not required, removed the constructor from the class.